financial-connections/src/main/java/com/stripe/android/financialconnections/features/generic/GenericScreen.kt

```diff
@@ -0,0 +1,345 @@
+                )
+            }
+        }
+
+        spacing?.let {
+            Spacer(modifier = Modifier.height(it))
+        }
+    }
+}
+
+@Composable
+internal fun GenericBody(
+    payload: Body,
+    modifier: Modifier = Modifier,
+    onClickableTextClick: (String) -> Unit,
+) {
+    Column(
+        modifier = modifier.fillMaxWidth(),
+    ) {
+        payload.entries.forEachIndexed { index, entry: Body.Entry ->
+            Box {
+                when (entry) {
+                    is Body.Entry.Image -> {
+                        StripeImage(
+                            url = entry.image.default.orEmpty(),
+                            contentDescription = entry.alt,
+                            imageLoader = LocalImageLoader.current,
+                            errorContent = { },
+                            modifier = Modifier.fillMaxWidth(),
+                        )
+                    }
+                    is Body.Entry.Text -> {
+                        val font = entry.size.toComposeSize()
+
+                        AnnotatedText(
+                            text = TextResource.Text(fromHtml(entry.text)),
+                            onClickableTextClick = onClickableTextClick,
+                            defaultStyle = font.copy(
+                                textAlign = entry.alignment.toComposeTextAlign()
+                            ),
+                            modifier = Modifier.padding(horizontal = 24.dp),
+                        )
+                    }
+                    else -> {
+                        Log.e("GenericBody", "Unsupported entry type: $entry")
+                    }
+                }
+            }
+
+            if (index != payload.entries.lastIndex) {
+                Spacer(modifier = Modifier.size(24.dp))
+            }
+        }
+    }
+}
+
+@Composable
+internal fun GenericHeader(
+    payload: Header,
+    onClickableTextClick: (String) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.fillMaxWidth(),
+    ) {
+        payload.icon?.default?.let { iconUrl ->
+            ShapedIcon(
+                modifier = Modifier.align(payload.alignment.toComposeAlignment()),
+                url = iconUrl,
+                contentDescription = null,
+                iconSize = if (payload.alignment == Alignment.Center) IconSize.Large else IconSize.Medium,
+            )
+            Spacer(modifier = Modifier.size(24.dp))
+        }
+
+        if (payload.title != null) {
+            AnnotatedText(
+                text = TextResource.Text(payload.title),
+                onClickableTextClick = onClickableTextClick,
+                defaultStyle = typography.headingXLarge.copy(
+                    textAlign = payload.alignment.toComposeTextAlign(),
+                ),
+                modifier = Modifier.fillMaxWidth(),
+            )
+        }
+
+        if (payload.subtitle != null) {
+            Spacer(modifier = Modifier.height(16.dp))
```

Comment:
I assume we’ll always have a title if we have a subtitle? Otherwise, we could also use `Arrangement.spacedBy()` here for consistent spacing.